README.md

```diff
@@ -95,36 +95,29 @@ _docs_ subdirectory. Version-specific documentation is also at
 
 ### Verifying binaries
 
-Download directories contain a `SHASUMS256.txt` file with SHA checksums for the
-files.
+Download directories contain a `SHASUMS256.txt.asc` file with SHA checksums for the
+files and the releaser PGP signature.
 
-To download `SHASUMS256.txt` using `curl`:
+You can get a trusted keyring from nodejs/release-keys, e.g. using `curl`:
 
 ```bash
-curl -O https://nodejs.org/dist/vx.y.z/SHASUMS256.txt
+curl -fsLo "/path/to/nodejs-keyring.kbx" "https://github.com/nodejs/release-keys/raw/HEAD/gpg/pubring.kbx"
 ```
 
-To check that downloaded files match the checksum, use `sha256sum`:
+Alternatively, you can import the releaser keys in your default keyring, see
+[Release keys](#release-keys) for commands to how to do that.
```

Comment:
The release keys section only contains commands to import individual keys from the `gpg-only-active-keys` set, so it is not a substitute for importing from `/gpg/pubring.kbx`

Maybe say here something like "you can import only the primary PGP releaser keys ..."

Reply:
I made a related proposal in https://github.com/nodejs/release-keys/issues/48 to have a keyring for supported releases only. That would make it simpler here. At the moment it would be out-of-scope for this PR which needs to deal with the keyrings as they are currently available.